src/components/booksearch/BookSearch.jsx

```diff
@@ -0,0 +1,65 @@
+import './booksearch.css';
+import books from '../../data/books.json';
+
+const BookSearch = () => {
+  return (
+   
+   
+   
+   
+   <main  className="booksearch__backgroundandscroll">
+
+
+
+
+
+
+
+    <div className="booksearch">
+
+      <div >
+        <h1>logo&apos;s space</h1>
+      </div>
+      <div className="booksearch__searcher">
+        <input className="booksearch__input" type="text" name="searcher" id="searcher" placeholder="Book's title" />
+
+
+        <button type="submit" className="booksearch__searchbutton">
```

Comment:
if you want to use 2 words as a className separate words by `-`
```suggestion
        <button type="submit" className="booksearch__search-button">
```